app/kumactl/cmd/root.go

```diff
@@ -62,8 +62,12 @@ func NewRootCmd(root *kumactl_cmd.RootContext) *cobra.Command {
 				}
 			}
 
-			if err := root.LoadConfig(); err != nil {
-				return err
+			if root.Args.ConfigType == kumactl_cmd.InMemory {
+				root.LoadInMemoryConfig()
+			} else {
+				if err := root.LoadConfig(); err != nil {
+					return err
+				}
```

Comment:
I think we can shorten it to:

```suggestion
				return root.LoadConfig()
```